### PR TITLE
Resolves #14

### LINKED ISSUE #14: Listing Show Page Image Consumption
- [x] Verify consumption of an image search API.
- [x] Create Service for Image Consumption.
  - [x] Add ability to add dominate color in the service request.
- [x] Create Facade for Image Consumption.
- [x] Create PORO for creating image instances.
- [x] Create View/feature to display the image on the Listing Show Page.

### PR DESCRIPTION
**Pull request by tjroeder**
User Story Issue #14
- Add Faraday for HTTP requests with rspec and spec.
- Add Image Service to connect to Google Image search for car images.
- Add Image Facade to create Image objects for display.
- Add Listings Controller to route requests to the listings#show page.
- Add Listing Show page with display of the car image.

### CODE REVIEW

File: app/facades/image_facade.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class ImageFacade
+  def self.search_for_listing_image(query, dom_color)
```
Comment (croixk) [inline app/facades/image_facade.rb:2]: I like the dominant color functionality

File: app/views/listings/show.html.erb [hunk 2]
```diff
@@ -0,0 +1 @@
+<%= image_tag @image.url, size: '600xauto', alt: @image.title %>
```
Comment (croixk) [inline app/views/listings/show.html.erb:1]: This seems like a good size for the image

**Review by croixk — APPROVED** [review]

File: app/services/image_service.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+class ImageService
+  def self.search_images(query, dom_color)
+    response = conn.get('v1', { q: query, imgColorType: 'color', imgDominantColor: dom_color, imgSize: 'LARGE', imgType: 'photo', searchType: 'image', num: 1 })
```
Comment (Eldridge-Turambi) [inline app/services/image_service.rb:3]: Thank you for doing this! This must have been tedious 💪

**Review by Eldridge-Turambi — APPROVED** [review]

**Comment by croixk** [comment]
Looks good - comments included

**Pull request merged**